MODULE.bazel

```diff
@@ -1,6 +1,18 @@
 module(name = "rules_ros")
 
 bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "boost.any", version = "1.83.0")
+bazel_dep(name = "boost.chrono", version = "1.83.0")
+bazel_dep(name = "boost.date_time", version = "1.83.0")
+bazel_dep(name = "boost.filesystem", version = "1.83.0")
+bazel_dep(name = "boost.interprocess", version = "1.83.0")
+bazel_dep(name = "boost.regex", version = "1.83.0")
+bazel_dep(name = "boost.scope_exit", version = "1.83.0")
+bazel_dep(name = "boost.signals2", version = "1.83.0")
```

Comment:
This is the only dep missing on BCR

Reply:
@hofbi `boost.signals2` is now available via BCR.

Reply:
Just found that `boost_program_options` is another dep 🙈

Reply:
adding [here](https://github.com/bazelbuild/bazel-central-registry/pull/2950)

Reply:
need more deps https://github.com/hofbi/rules_ros/pull/1

Reply:
Awesome, all deps on BCR and CI is green 🎉